src/app/components/cron-gen/cron-gen.jsx

```diff
@@ -40,13 +42,48 @@ const useStyles = makeStyles({
       justifyContent: 'space-between',
     },
   },
+  cronValue: {
```

Comment:
With the change to show the english version of the schedule, we don't need to show the cron value at all; that's why I deleted it in my branch